specification/cognitiveservices/OpenAI.Inference/models/images.tsp

```diff
@@ -0,0 +1,137 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+
+using Azure.Core;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024.")
+enum ImageSize {
+  @doc("Small image size (256x256).")
+  Small: "256x256",
+
+  @doc("Medium image size (512x512).")
+  Medium: "512x512",
+
+  @doc("Large image size (1024x1024).")
+  Large: "1024x1024",
```

Comment:
I'd prefer to be explicit about the sizes because I somehow suspect that Large will not stay the biggest value forever.

Reply:
yeah, couldn't think of a name that would actually work with the generators though (that's why I changed it)... when I tried `256x256`  for enum name C# generated something like `ImageSize._256x256` for the value to be a valid ID

Reply:
I believe using a prefix like `Size` (or even repeating the "Image" part would be preferable. E.g. `ImageSize.Size256x256` or `ImageSize.ImageSize256x256`.

Reply:
@johanste do you have a recommendation on a way we could keep this pretty across languages? I agree with you on the fragility of the generic labels; I'd love if we could just encode this as proper width and height parameters that we combine ourselves if needed, but that comes with its own drawbacks with how easily invalid input becomes to provide.

We could have a middle ground of something like "Size256x256" -- that just feels a little bit ugly.

Reply:
Across all languages? Not sure - but if I was expressing this in a TypeScript library, I'd probably prefer to use a union and completely ignore the labels. Same for Python. And thus avoiding the name/enumness altogether.

Reply:
updated with that